test/e2e/e2e.sh

```diff
@@ -258,7 +258,7 @@ function e2e::main() {
   shell::spawn e2e::run_with_default
 
   # Run tests when open kruise is enabled.
-  shell::spawn e2e::run_with_open_kruise
+  # shell::spawn e2e::run_with_open_kruise
```

Comment:
I wonder, in the previous time, why we need to run `run_with_default` and `run_with_open_kruise`

Reply:
Because the underlying workload objects are different and I would like to see if they both works. But it turns out running them sequentially doubles the time while running them in parallel will make the test flaky. So I decided to disable one for now.